cmd/serverutil/serverutil.go

```diff
@@ -41,23 +42,31 @@ func GrpcHandlerFunc(grpcServer http.Handler, otherHandler http.Handler) http.Ha
 	})
 }
 
-// RegisterServiceFromEndpoint registers services with a grpc server's ServeMux
-type RegisterServiceFromEndpoint func(context.Context, *runtime.ServeMux, string, []grpc.DialOption) error
+// RegisterServiceFromConn registers services with a grpc server's ServeMux
+type RegisterServiceFromConn func(context.Context, *runtime.ServeMux, *grpc.ClientConn) error
+
+// ServeAPIGatewayAndGRPC serves
```

Comment:
Looks like this comment is only half finished.

Reply:
sorry!